sca-wrapper-play-30/src/main/scala/uk/gov/hmrc/sca/models/BannerConfig.scala

```diff
@@ -16,4 +16,9 @@
 
 package uk.gov.hmrc.sca.models
 
-case class BannerConfig(showAlphaBanner: Boolean, showBetaBanner: Boolean, showHelpImproveBanner: Boolean)
+case class BannerConfig(
+  showAlphaBanner: Boolean,
+  showBetaBanner: Boolean,
+  showHelpImproveBanner: Boolean,
```

Comment:
Mark this parameter as deprecated. the preferred method for the UR banner is to use the config in single-customer-account-wrapper-data
https://github.com/hmrc/app-config-base/blob/198ca49963d7a5be6f8d83084f84d9827d4c6cc1/single-customer-account-wrapper-data.conf#L53